admin/controllers/pupil.js

```diff
@@ -62,6 +61,8 @@ const postAddPupil = async (req, res, next) => {
   res.locals.pageTitle = 'Add pupil'
   try {
     const pupil = await pupilAddService.addPupil(req.body, req.user.schoolId)
+    const pupilRegisterRedisKey = `pupilRegisterViewData:${req.user.schoolId}`
+    await redisCacheService.drop(pupilRegisterRedisKey)
```

Comment:
we should be invalidating cache in the service, not the controller

Reply:
@GuyHarwood updated